doc/classes/Input.xml

```diff
@@ -171,6 +172,7 @@
 				Gets an input vector by specifying four actions for the positive and negative X and Y axes.
 				This method is useful when getting vector input, such as from a joystick, directional pad, arrows, or WASD. The vector has its length limited to 1 and has a circular deadzone, which is useful for using vector input as movement.
 				By default, the deadzone is automatically calculated from the average of the action deadzones. However, you can override the deadzone to be whatever you want (on the range of 0 to 1).
+				[b]Note:[/b] This reflects the global input state and is not affected by [method Control.accept_event] or [method Viewport.set_input_as_handled], which only deal with the way input is propagated in the [SceneTree].
```

Comment:
Isn't the note in the description enough? Repeating it in every single method seems a bit too redundant. The main description isn't too big so users should be expected to read it to know the global context of this class.

Reply:
That's what I originally intended, can revert it if it's considered enough there

Reply:
Done